index.html

```diff
@@ -4,9 +4,71 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="style.css" />
     <title>Anwar Hussaini's Portfolio</title>
   </head>
   <body>
-    <h1 class="heading-primary">My Portfolio</h1>
+    <header class="main-header bg-color">
+      <div class="notification-bar">
+        <img src="Light.png" alt="Notification Bar" class="notification-img" />
+      </div>
+      <div class="container">
+        <nav class="main-nav">
+          <div class="logo-box"></div>
+          <div class="menus-box"><i class="fa-solid fa-bars menu icon"></i></div>
+        </nav>
+      </div>
+    </header>
+
+    <main>
+      <section class="section-hero bg-color">
+        <div class="container">
+          <h1 class="heading-primary">
+            <div class="hero-name">
+              <span>Hey There.</span>
+              <span>I'm Anwar</span>
+            </div>
+            <div class="hero-job">
+              <span>I'm a software</span>
+              <span>developer</span>
+            </div>
+          </h1>
+
+          <div class="hero-description">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </div>
+
+          <div class="social-media-icons">
+            <ul class="icons">
+              <li><i class="fa-brands fa-twitter icon"></i></li>
+              <li>
+                <i class="fa-brands fa-linkedin-in icon"></i>
+              </li>
+              <li><i class="fa-solid fa-m icon"></i></li>
+              <li><i class="fa-brands fa-github icon"></i></li>
+              <li><i class="fa-brands fa-angellist icon"></i></li>
+            </ul>
```

Comment:
- [ ] Please stick to the Figma design your icons are not the same as the Figma design so kindly replace these icons with the Figma design icons :slightly_smiling_face:  check below :point_down:  the comparison between your design and Figma design :+1: 

| Your Design | Figma Design |
|--------|--------|
| ![Screenshot from 2023-04-27 15-59-49](https://user-images.githubusercontent.com/45226848/234847758-9d68a43a-3177-46df-aec0-7c7956248b0c.png) | ![Screenshot from 2023-04-27 15-59-43](https://user-images.githubusercontent.com/45226848/234847782-dadeab0b-6530-4136-8a8b-27337f5d4769.png) | 

- [ ] Also each icon has a link (`a` tag).